domains/DataCollection/Forms/SupplementaryForm/configs/envhealth.config.js

```diff
@@ -208,36 +274,57 @@ const configArray = {
     options: [
       {
         label: 'global.yes',
-        value: 'Yes'
+        value: 'Y'
       },
       {
         label: 'global.no',
-        value: 'No'
+        value: 'N'
       }
     ],
     validation: false
   },
   {
     label: 'environmentalHealth.floorCondition.label',
-    formikKey: 'conditionoFloorinyourhouse',
+    formikKey: 'floorCondition',
```

Comment:
why are we changing the formikKey? all other values collected will be in separate field